packages/metrics/jest.config.js

```diff
@@ -0,0 +1 @@
+module.exports = require("@k8slens/jest").monorepoPackageConfig(__dirname).configForReact;
```

Comment:
Should this be in the dependencies or does it just work?

Reply:
added

Reply:
Should be added yeah. Hard to remember because they seem to work anyway because of node's module resolution and `npm` workspaces automatically hoisting dependencies to root of monorepo. If we had something more strict, like `pnpm` then it would be easier to remember.

Reply:
This PR makes it bit more strict #7545 to remind us.